<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="redView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="blueView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="yellowView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="greenView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,15,1">
      <item>
       <widget class="QLabel" name="Turn">
        <property name="text">
         <string>Turn: 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="BoardView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newGameButton">
          <property name="text">
           <string>New Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
